File: frontend/staff-web-app/src/models/refund/RefundRequest.js
class RefundRequest {
  refundMode;
  reason;
  refundLineItemRequests;
  customerId;

  constructor(refundMode, reason, refundLineItemRequests, customerId) {
    this.refundMode = refundMode;
    this.reason = reason;
    this.refundLineItemRequests = refundLineItemRequests;
    this.customerId = customerId;
  }
}

export default RefundRequest;

File: backend/src/main/java/capstone/rt04/retailbackend/request/refund/RefundRequest.java
package capstone.rt04.retailbackend.request.refund;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.util.List;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class RefundRequest {
    private String refundMode;
    private String reason;
    private Long customerId;
    private List<RefundLineItemRequest> refundLineItemRequests;
}
